### PR TITLE
(#5102) - Use LTS node.js for now

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

**Comment by nolanlawson** [comment]
:+1: when green

File: .travis.yml [hunk 1]
```diff
@@ -137,10 +137,19 @@ matrix:
   - env: COMMAND=report-coverage
   - env: CLIENT=selenium:firefox:41.0.1 SERVER=couchdb-master COUCH_HOST=https://$CLOUDANT_USERNAME:$CLOUDANT_PASSWORD@pouch.cloudant.com COMMAND=test
   - env: CLIENT="saucelabs:MicrosoftEdge" COMMAND=test
+  - node_js: "stable"
+    services: docker
+    env: CLIENT=node COMMAND=test
+  - node_js: "0.10"
+    services: docker
+    env: CLIENT=node COMMAND=test
```
Comment (nolanlawson) [inline .travis.yml]: ahh, so this is how you add 0.10 to allowed failures. LGTM

**Comment by daleharvey** [comment]
Checking up on why these arent green now

Yeh I figured if I was putting stable in expected failures may as well do .10 at the same time

**Comment by daleharvey** [comment]
So confused at why this hasnt broken before, but we require('body-parser') in the tests but dont list it in package.json, also this only seemed to break locally but running `test-component` wipes the lib directory then never rebuilds extras, so just switched it to full build

**Pull request merged**